src/treelist.c

```diff
@@ -2489,6 +2489,9 @@ treelist_insert_column(treelist_t* tl, int col_ix, const MC_TLCOLUMN* col,
         return -1;
     }
 
+	/* Implement MC_TLCF_IMAGE by sending HDM_SETIMAGELIST to show icon on WC_HEADER .iImage*/
+	MC_SEND(tl->header_win, HDM_SETIMAGELIST, (WPARAM)HDSIL_NORMAL, (LPARAM)tl->imglist);
```

Comment:
1. Please use spaces instead of tabs for indentation, as in the rest of the source file.
2. I think it should rather be in `treelist_set_imagelist()` so we are sure that the treelist itself and the header always use the same image list handle. It's application who manages lifetime of the image list. And if the application later calls `treelist_set_imagelist()` to reset it, it should be allowed to destroy the old image list without a danger of crash.

Reply:
1: I think VS2019 auto uses tabs for indentation.
2: You're right, best place to call it is in treelist_set_imagelist().